ferrilata-devilline/Models/Holder.cs

```diff
@@ -0,0 +1,16 @@
+namespace ferrilata_devilline.Models
+{
+    public class Holder
+    {
```

Comment:
i don't get why do we need this instead of an anon object, but that does not mean we don't.